### ISSUE TITLE
auto-service-ksp raise error while processing java source

### ISSUE THREAD

**Issue by ganadist** [ZacSweers/auto-service-ksp#18]
How to reproduce

* fetch source
```
$ git clone https://github.com/ganadist/VersionCodeDemo -b autoservice_ksp_test test
$ cd test
```

* compile failed while using `auto-service-ksp`
```
$ ./gradlew :app:assembleDebug

> Task :app:kspBetaDebugKotlin FAILED
e: [ksp] test/app/src/main/java/com/example/myapplication/TestServiceImpl.java:5: No 'value' member value found!

```

* compile passed while using `kapt`
```
$ git checkout HEAD~1
$ ./gradlew :app:assembleDebug

```

**Comment by ganadist** [comment]
In this case, `KSValueArgument` was found by `annotation.arguments.find { it.name?.getShortName() == "value" }` , but `value` field is `null`
https://github.com/ZacSweers/auto-service-ksp/blob/main/processor/src/main/kotlin/dev/zacsweers/autoservice/ksp/AutoServiceSymbolProcessor.kt#L86

Is this a bug of ksp?

**Comment by ZacSweers** [comment]
This is now fixed and released in 0.5.5. Sorry again for the delay and thank you for the fix!
